op-challenger/challenger/challenger.go

```diff
@@ -16,11 +16,15 @@ import (
 	metrics "github.com/ethereum-optimism/optimism/op-challenger/metrics"
 
 	bindings "github.com/ethereum-optimism/optimism/op-bindings/bindings"
-	sources "github.com/ethereum-optimism/optimism/op-node/sources"
+	eth "github.com/ethereum-optimism/optimism/op-node/eth"
 	opclient "github.com/ethereum-optimism/optimism/op-service/client"
```

Comment:
its a bit redundant to use named imports for these

Reply:
I know, it's just personal preference to make it easier to read since explicit imports are all aligned at the start of the line rather than the end of a url.